app/controllers/index.js

```diff
@@ -0,0 +1,29 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import ENV from 'website-my/config/environment';
+
+const BASE_URL = ENV.BASE_API_URL;
+
+export default class IndexController extends Controller {
+  @tracked status = this.model;
+
+  @action async updateStatus(status) {
+    this.status = status;
+    try {
+      const response = await fetch(`${BASE_URL}/users/self`, {
```

Comment:
Curious why the fetching of data is happening in the controller and not the route. Also are we not using Ember Data?